jekyll/_cci2_ja/collect-test-data.md

```diff
@@ -447,9 +447,6 @@ Use [trx2junit](https://github.com/gfoidl/trx2junit) to convert Visual Studio /
 
 A working `.circleci/config.yml` section might look like this:
 
-```yaml
-    A working `.circleci/config.yml` section might look like this:
-
```

Comment:
I already have a PR for that: https://github.com/circleci/circleci-docs/pull/5946

This will disappear from here once this is merged.